src/scss/_content.scss

```diff
@@ -1,46 +1,47 @@
-// sass-lint:disable no-qualifying-elements
-
 .o-techdocs-content {
 	@include oTechdocsFont();
+	@include oNormaliseText();
 
-	h1,
-	h2,
-	h3,
-	h4,
-	h5,
-	h6,
-	p {
-		font-weight: 300;
-	}
 	h1 {
-		font-size: 34px;
+		font-size: 30px;
+		line-height: oTechdocsSpacingUnit(4);
+		padding-bottom: oTechdocsSpacingUnit(2);
+		margin-bottom: oTechdocsSpacingUnit(2);
+		@include oGridRespondTo(M) {
+			font-size: 40px;
+			line-height: oTechdocsSpacingUnit(6);
+			padding-bottom: oTechdocsSpacingUnit(3);
+			margin-bottom: oTechdocsSpacingUnit(3);
+		}
 	}
 	h2 {
+		line-height: oTechdocsSpacingUnit(4);
 		font-size: 26px;
-		border-bottom: 1px solid $o-techdocs-color-grey2;
-		padding-bottom: 2px;
-		margin: 35px 0 15px;
+		padding-bottom: 4px; // magic number to make padding + border-bottom add up to 8 (2*spacing unit)
```

Comment:
Is it possible to use `calc` here? It might help avoid the use of a magic number.